core/DataTable/Filter/ColumnCallbackAddMetadata.php

```diff
@@ -45,10 +45,6 @@ public function __construct($table, $columnsToRead, $metadataToAdd, $functionToA
     {
         parent::__construct($table);
 
-        if (!is_array($columnsToRead)) {
-            $columnsToRead = array($columnsToRead);
-        }
-
         $this->columnsToRead      = $columnsToRead;
```

Comment:
Could you tell me how much of an improvement handling the single columnsToRead case has? Also, does this optimize something in MultiSites.getAll?